.env.example

```
@@ -0,0 +1,122 @@
+# ===================================
+# Firecrawl Environment Configuration
+# ===================================
+# This is an example configuration file. Copy to .env and update values as needed.
+# Sensitive values should be stored in .env.local (not tracked in git).
+
+# ===== Core Service Configuration =====
+NUM_WORKERS_PER_QUEUE=16
+PORT=3002
+HOST=0.0.0.0
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The HOST key should go before the NUM_WORKERS_PER_QUEUE key

(UnorderedKey)